src/custom_modules/DAO.ts

```diff
@@ -60,19 +59,23 @@ export const DAO = {
    * @param {function} cb Callback for when DAO has been sucessfully set up
    */
   connect: async function (cb) {
-    const dbSocketPath = process.env.DB_SOCKET_PATH || "/cloudsql";
-
-    var args = {
+    const args = {
       user: config.db_username,
       password: config.db_password,
       database: config.db_name,
       waitForConnections: true,
       enableKeepAlive: true as true, // Workaround for type checking quirk
       timezone: "+00:00",
     };
-    if (process.env.K_SERVICE != null)
-      args["socketPath"] = `${dbSocketPath}/${process.env.CLOUD_SQL_CONNECTION_NAME}`;
-    else args["host"] = "127.0.0.1";
+
+    // Using Unix sockets to connect to cloud SQL is the default way
+    // https://cloud.google.com/sql/docs/mysql/connect-run#connect-unix-socket
+    if (process.env.K_SERVICE != null) {
+      args["socketPath"] = `cloudsql/${process.env.CLOUD_SQL_CONNECTION_NAME}`;
```

Comment:
Does it matter if the socketPath does not have a leading `/` after this refactoring?

Reply:
Yes :grimacing: Added it